omnibus/config/projects/agent.rb

```diff
@@ -205,6 +205,31 @@
     'Platform' => "#{arch}",
     'IncludeSysprobe' => "#{include_sysprobe}",
   })
+  # This block runs before harvesting with heat.exe
+  # It runs in the scope of the packager, so all variables access are from the point-of-view of the packager.
+  # Therefore, `install_dir` does not refer to the `install_dir` of the Project but that of the Packager.
+  pre_heat do
+    def generate_embedded_archive(version)
+      safe_embedded_path = windows_safe_path(install_dir, "embedded#{version}")
+      safe_embedded_archive_path = windows_safe_path(install_dir, "embedded#{version}.7z")
+
+      shellout!(
+        <<-EOH.split.join(" ").squeeze(" ").strip
```

Comment:
Nitpicking here, but is the `squeeze` safe? What if `#{safe_embedded_archive_path}` was `"C:\some path"` with multiple spaces between `some` and `path`? I think leaving multiple spaces should do no harm, right?

Reply:
Well I didn't know there were multiple occurrences. I think starting with not adding more is good enough for now.